python/graphstorm/dataloading/dataset.py

```diff
@@ -26,9 +26,11 @@
 from torch.utils.data import Dataset
 import pandas as pd
 
-from ..utils import get_rank, get_world_size, is_distributed, barrier
-from ..utils import sys_tracker, is_wholegraph
-from .utils import dist_sum, flip_node_mask, is_wholegraph_embedding
+from ..utils import get_rank, get_world_size, is_distributed, barrier, is_wholegraph
+from ..utils import sys_tracker
+from .utils import dist_sum, flip_node_mask
+
+from ..wholegraph_utils import is_wholegraph_embedding
```

Comment:
Can we have a whole graph sub-module under graphstorm?
Then we can do `from ..wholegraph import xxx` (or say `from graphstorm.wholegraph import xxx`) and `from ..wholegraph .utils import xxx`.

Reply:
Moved wholegraph-related functions/classes to a sub-package `graphstorm.wholegraph`. Is this what you expected?